invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/config/index.js

```diff
@@ -62,7 +62,8 @@ const resultsPerPageValues = [
 const logo = "/static/images/logo.svg";
 
 const searchApi = {
-  url: "https://localhost:5000/api/records/",
+  baseURL: "",
+  url: "/api/records",
```

Comment:
Tested the `containerize` and it works ;)

Reply:
nice!